README.md

```diff
@@ -57,11 +57,12 @@ This is [Acra Community Edition](https://www.cossacklabs.com/acra/#pricing), it'
 </tr><tr><td><li>Trust compartmentalisation </li></td><td> datastore and application components can be compromised, yet the data stays protected</td>
 </tr><tr><td><li>Prevention of SQL injections </li></td><td> through a built-in SQL firewall </td>
 </tr><tr><td><li>Intrusion detection system </li></td><td> using poison records (honey tokens) to warn about suspicious behaviour </td>
-</tr><tr><td><li>Searchable encryption ᵉ </li></td><td rowspan=3>available for <a href="https://www.cossacklabs.com/acra/#pricing" target="_blank">Acra Enterprise</a> users</td>
-</tr><tr><td><li>Masking ᵉ </li></td>
-</tr><tr><td><li>Obfuscation / tokenization ᵉ </li></td>
-</tr><tr><td><li>Pseudonymisation </li></td><td rowspan=2> <i>coming in the nearest releases</i>
-</tr><tr><td><li>Cryptographically protected audit log </li></td>
+</tr><tr><td><li>REST API (gRPC, HTTP) </li></td><td> encryption/decryption using AcraTranslator </td>
+</tr><tr><td><li>Searchable encryption ᵉ </li></td><td rowspan=5>available for <a href="https://www.cossacklabs.com/acra/#pricing" target="_blank">Acra Enterprise</a> users</td>
+</tr><tr><td><li>Masking / anonymization ᵉ </li></td>
+</tr><tr><td><li>Tokenization / Pseudonymization ᵉ </li></td>
+</tr><tr><td><li>Cryptographically protected audit log ᵉ </li></td>
+</tr><tr><td><li>Key rotation without data re-encryption ᵉ </li></td>
```

Comment:
IIRC, it is possible to rotate just keys in Acra CE too, it's not a EE-exclusive feature.

Reply:
🐱

Reply:
fxd